docs/writing-rules/rule-syntax.md

```diff
@@ -121,6 +132,28 @@ This operator will filter findings that have _any overlap_ with the supplied reg
 
 <iframe src="https://semgrep.dev/embed/editor?snippet=8n5Q" border="0" frameBorder="0" width="100%" height="432"></iframe>
 
+### `focus-metavariable`
+
+The `focus-metavariable` operator puts the focus, or _zooms in_, on the code region matched by a metavariable. For example, to find all functions arguments annotated with the type `bad` you may write the following pattern:
+
+```yaml
+pattern: |
+  def $FUNC(..., $ARG : bad, ...):
+    ...
+```
+
+This works but it matches the entire function definition. Sometimes, this is not desirable. In particular, if you are using [Semgrep App](https://semgrep.dev/login) and you have triaged a finding generated by this pattern, the same finding will show up again as new if you make any change to the definition of the function!
```

Comment:
@IagoAbal Just to specify why this is beneficial we can add more explanation here. Let me know if what I wrote is correct... feel free to edit my suggestions.

```suggestion
This works but it matches the entire function definition. Sometimes, this is not desirable. If the definition spans hundreds of lines, they are all matched. In particular, if you are using [Semgrep App](https://semgrep.dev/login) and you have triaged a finding generated by this pattern, the same finding shows up again as new if you make any change to the definition of the function!
```